src/python-common/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+if(WITH_PYTHON2)
+  set(python_version_major 2)
+else()
+  set(python_version_major 3)
+endif()
+
+include(Distutils)
+distutils_install_module(ceph
+  INSTALL_SCRIPT ${CMAKE_INSTALL_FULL_SBINDIR}
+  PYTHON_VERSION ${python_version_major})
```

Comment:
> Which is expected, as Python 3 was not enabled. Is there a way to disable the creation of the python 3 debian package, if Python 3 is not enabled?
actually, python3 is enabled when building debian packages, see 

https://github.com/ceph/ceph/blob/633a3fa8aee57676ebe4e461d33853fd820403dc/debian/rules#L21

for building both python2 and python3 packages, you just need to run `distutils_install_module()` twice. one for python2, and another for python3.